js/lib/d3-timeline.js

```diff
@@ -537,6 +538,16 @@
       width = w;
       return timeline;
     };
+	
+	timeline.groupCount = function (times, slack) {
+      if (!arguments.length) return groupCount;
+	   //The following call to groupByOverlap gives error
+	  //var = groups = groupByOverlap(times, slack);
+      //groupCount = groups.length;
```

Comment:
To make this work you need to move groupByOverlap outside the `timeline(gParent)` function. Then it works fine.